From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Venelin <venelin@pulumi.com>
Date: Fri, 10 May 2024 18:03:03 +0100
Subject: [PATCH] fixup! update CHANGELOG.md v3.8.1


diff --git a/go.mod b/go.mod
index 98a5fd37..a2c9b372 100644
--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,4 @@
-module github.com/PagerDuty/terraform-provider-pagerduty
+module github.com/PagerDuty/terraform-provider-pagerduty/v3
 
 go 1.20
 
diff --git a/main.go b/main.go
index b8782ded..b57b09ba 100644
--- a/main.go
+++ b/main.go
@@ -8,8 +8,8 @@ import (
 	"github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server"
 	"github.com/hashicorp/terraform-plugin-mux/tf5muxserver"
 
-	"github.com/PagerDuty/terraform-provider-pagerduty/pagerduty"
-	pagerdutyplugin "github.com/PagerDuty/terraform-provider-pagerduty/pagerdutyplugin"
+	"github.com/PagerDuty/terraform-provider-pagerduty/v3/pagerduty"
+	pagerdutyplugin "github.com/PagerDuty/terraform-provider-pagerduty/v3/pagerdutyplugin"
 )
 
 func main() {
diff --git a/pagerduty/util.go b/pagerduty/util.go
index 4b0dcc1e..923000cd 100644
--- a/pagerduty/util.go
+++ b/pagerduty/util.go
@@ -6,8 +6,9 @@ package pagerduty
 import (
 	"time"
 
-	"github.com/PagerDuty/terraform-provider-pagerduty/util"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/PagerDuty/terraform-provider-pagerduty/v3/util"
 )
 
 func timeToUTC(v string) (time.Time, error) {
diff --git a/pagerdutyplugin/config.go b/pagerdutyplugin/config.go
index 9fc588e0..a0f10f7e 100644
--- a/pagerdutyplugin/config.go
+++ b/pagerdutyplugin/config.go
@@ -12,9 +12,10 @@ import (
 	"time"
 
 	"github.com/PagerDuty/go-pagerduty"
-	"github.com/PagerDuty/terraform-provider-pagerduty/util"
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/logging"
+
+	"github.com/PagerDuty/terraform-provider-pagerduty/v3/util"
 )
 
 // Config defines the configuration options for the PagerDuty client
diff --git a/pagerdutyplugin/provider_test.go b/pagerdutyplugin/provider_test.go
index f562e13b..eef50782 100644
--- a/pagerdutyplugin/provider_test.go
+++ b/pagerdutyplugin/provider_test.go
@@ -6,13 +6,14 @@ import (
 	"testing"
 	"time"
 
-	pd "github.com/PagerDuty/terraform-provider-pagerduty/pagerduty"
 	"github.com/PagerDuty/terraform-provider-pagerduty/util"
 	"github.com/hashicorp/terraform-plugin-framework/providerserver"
 	"github.com/hashicorp/terraform-plugin-go/tfprotov5"
 	"github.com/hashicorp/terraform-plugin-mux/tf5muxserver"
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	pd "github.com/PagerDuty/terraform-provider-pagerduty/v3/pagerduty"
 )
 
 var testAccProvider = New()
